index.html

```diff
@@ -652,26 +652,26 @@ <h3>
         </h3>
         <p>
           To <dfn data-lt="prompt the user to choose|prompting the user to choose" class=
-          "export">prompt the user to choose</dfn> one of several |options| associated with a
-          |descriptor|, the user agent must perform the following steps. This algorithm returns
-          either {{PermissionState/"denied"}} or one of the options.
+          "export">prompt the user to choose</dfn> one or more of several |options| associated with
+          a |descriptor|, the user agent must perform the following steps. This algorithm returns
```

Comment:
```suggestion
          a |descriptor|, and optionally a [=non-negative integer=] greater than 1 |restrictions|, which restricts how many |options| a user can select, the user agent must perform the following steps. This algorithm returns
```

Reply:
FWIW, I'm not sure any callers are going to pick options other than "1" or "unlimited". Those are also the options for `<input>` elements with the [`multiple`](https://html.spec.whatwg.org/multipage/input.html#attr-input-multiple) attribute, which maybe reinforces the idea that they're all we should support here.

Reply:
Select multiple allows for arbitrary option selection though, if you use the modifier keys (alt/cmd, shift, cmd/ctrl-a, etc): https://miketaylr.com/misc/select-multiple.html

Reply:
Sorry, I meant that `<input>` allows the caller (the HTML author) to restrict the user to picking 1 option (omit `multiple`) or an unlimited number of options (include `multiple`), but not to restrict the user to picking up to 5 options. It'd be plausible to do the same here.

Reply:
Aha 💡🧠  - yeah, sounds reasonable.